src/vellum/workflows/workflows/base.py

```diff
@@ -602,7 +607,7 @@ def _generate_filtered_events() -> Generator[BaseWorkflow.WorkflowEvent, None, N
                 if should_yield(self.__class__, event):
                     yield event
 
-        return WorkflowEventGenerator(_generate_filtered_events(), runner_stream.span_id)
+        return WorkflowEventGenerator(_generate_filtered_events(), runner_stream.span_id, event_max=event_max)
```

Comment:
DevinAI we should pass this to the WorkflowRunner so that we could guarantee that the workflow events in the VellumEmitter also experience redaction